pkg/controller/informers/extensions.go

```diff
@@ -68,3 +68,46 @@ func (f *daemonSetInformer) Lister() *cache.StoreToDaemonSetLister {
 	informer := f.Informer()
 	return &cache.StoreToDaemonSetLister{Store: informer.GetIndexer()}
 }
+
+// Deployment is type of SharedIndexInformer which watches and lists all pods.
+// Interface provides constructor for informer and lister for pods
```

Comment:
Did you intend to write "Interface" here, as in "the interface", or did you mean "Deployment"?